packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -78,4 +98,176 @@ describe('test project paths', () => {
+      const apexTestResultsFolder = projectPaths.apexTestResultsFolder();
+      expect(apexTestResultsFolder).toEqual(
+        path.join(FAKE_WORKSPACE, TOOLS, TEST_RESULTS, APEX)
+      );
+    });
+  });
+
+  describe('test apexLanguageServerDatabase', () => {
+    let stateFolderStub: jest.SpyInstance;
+
+    beforeEach(() => {
+      stateFolderStub = jest
+        .spyOn(projectPaths, 'stateFolder')
+        .mockReturnValue(FAKE_WORKSPACE);
+    });
+    it('should be defined', () => {
+      expect(projectPaths.apexLanguageServerDatabase).toBeDefined();
+    });
+
+    it('should return a path to the apex Language Server Database folder based on root workspace', () => {
+      const apexLanguageServerDatabase = projectPaths.apexLanguageServerDatabase();
+      expect(apexLanguageServerDatabase).toEqual(
+        path.join(FAKE_WORKSPACE, TOOLS, APEX_DB)
+      );
+    });
+  });
+
+  describe('test lwcTestResultsFolder', () => {
+    let stateFolderStub: jest.SpyInstance;
+
+    beforeEach(() => {
+      stateFolderStub = jest
+        .spyOn(projectPaths, 'stateFolder')
+        .mockReturnValue(FAKE_WORKSPACE);
+    });
+    it('should be defined', () => {
+      expect(projectPaths.lwcTestResultsFolder).toBeDefined();
+    });
+
+    it('should return a path to the lwc Test Results Folder based on root workspace', () => {
+      const lwcTestResultsFolder = projectPaths.lwcTestResultsFolder();
+      expect(lwcTestResultsFolder).toEqual(
+        path.join(FAKE_WORKSPACE, TOOLS, TEST_RESULTS, LWC)
+      );
+    });
+  });
+
+  describe('test testResultsFolder', () => {
+    let stateFolderStub: jest.SpyInstance;
+
+    beforeEach(() => {
+      stateFolderStub = jest
+        .spyOn(projectPaths, 'stateFolder')
+        .mockReturnValue(FAKE_WORKSPACE);
+    });
+    it('should be defined', () => {
+      expect(projectPaths.testResultsFolder).toBeDefined();
+    });
+
+    it('should return a path to the  Test Results Folder based on root workspace', () => {
+      const testResultsFolder = projectPaths.testResultsFolder();
+      expect(testResultsFolder).toEqual(
+        path.join(FAKE_WORKSPACE, TOOLS, TEST_RESULTS)
+      );
+    });
+  });
+
+  describe('test debugLogsFolder', () => {
+    let stateFolderStub: jest.SpyInstance;
+
+    beforeEach(() => {
+      stateFolderStub = jest
+        .spyOn(projectPaths, 'stateFolder')
+        .mockReturnValue(FAKE_WORKSPACE);
+    });
+    it('should be defined', () => {
+      expect(projectPaths.debugLogsFolder).toBeDefined();
+    });
+
+    it('should return a path to the debug Logs folder based on root workspace', () => {
+      const debugLogsFolder = projectPaths.debugLogsFolder();
+      expect(debugLogsFolder).toEqual(
+        path.join(FAKE_WORKSPACE, TOOLS, DEBUG, LOGS)
+      );
+    });
+  });
+
+  describe('test fileExtensionsMatch IN PROGRESS', () => {
+    const mockEditor = {
+      document: {
+        uri: { path: 'foo.log' } as vscode.Uri
```

Comment:
Ok so the tricky part here is you have to keep in mind that all things vscode.* in jest tests are mocked.  So you were calling a jest mock when you called vscode.Uri.file and it was just resolving nothing.